win32ss/user/ntuser/timer.c

```diff
@@ -219,8 +221,12 @@ IntSetTimer( PWND Window,
   {
       IntLockWindowlessTimerBitmap();
 
-      IDEvent = RtlFindClearBitsAndSet(&WindowLessTimersBitMap, 1, HintIndex);
-
+      IDEvent = RtlFindClearBitsAndSet(&WindowLessTimersBitMap, 1, HintIndex++);
```

Comment:
Realistically it comes out about the same, because the returned ID values form a continuous series. Windows returns 0xffff, 0xfffe, 0xfffd ... (i.e. index 1,2,3). Moreover, if it theoretically ran in multiple threads, some index could be freed in the meantime. For me, there's no point in changing this unless it would nerrally save some instructions.